Source/Editor/DualityEditor/Utility/DualityEditorUserData.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using Duality.Resources;
+
+namespace Duality.Editor
+{
+	public class DualityEditorUserData
```

Comment:
Needs XML docs on all public API!

Reply:
Iam not sure what kind of xml documentation I could write here for all the members.

If its just 'Gets or Sets x' then I rather leave it empty.

Reply:
Examples, not fleshed out properly:

- `Backups`: Specifies whether the editor will copy previous versions of saved resources to a local `Backups` folder.
- `AutoSaves`: Specifies auto-save behavior in the editor, e.g. the frequency at which unsaved Resources are saved.
- `FirstSession`: Whether this is the first time the editor has been started in this project.

And so on. Context and examples can help a lot! It's also about understanding what those values do conceptually, not just about what the property does (get / set).

If you want, I can help out on this one and join in with a few commits on the PR, but give me a few days in that case.

Reply:
If you can jump in and help out that would be great. Iam a bit time constrained atm.

Reply:
Will do 👍